### PR TITLE
Homeandgamenewverion

### LINKED ISSUE #11: Game Page
- [ ] bulid game page structure
- [ ] move from  popup to another
- [ ] styling game page 
- [ ] get  the level data from database
- [ ] drag and drop blockes
- [ ] check the solution when click on play button  #8  
- [ ] render the navigation and footer #15 #14

### PR DESCRIPTION
**Pull request by ismail2009**
### Fixes
fixes #11 #5 #14 

### Related
related to #11 #5 #14 
### Summary
redesign homepage and game page with page responsive

### CODE REVIEW

File: public/js/dom.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  ev.dataTransfer.setData('text', ev.target.id);
+}
+function allowDrop(ev) {  // eslint-disable-line
+  ev.preventDefault();
+}
+function dropHandle(ev) { // eslint-disable-line
+  ev.preventDefault();
+  var  data = ev.dataTransfer.getData('text'); // eslint-disable-line
+  ev.target.appendChild(document.getElementById(data)); // eslint-disable-line
+  console.log('hghg');
```
Comment (NadiaKhatib) [inline public/js/dom.js]: Delete this line console.log('hghg');

File: public/css/style.css [hunk 2]
```diff
@@ -103,3 +103,36 @@ html,body{
...
+  min-width: 40px;
+  padding: 2em 0.5em;
+  color: #fff;
+  background-color: #29e;
+  border: 2px solid #fff;
+  transform: translate(0,0);
+  transition: background-color 0.3s;
+  height: 30px;
+}
+/*.drag-drop .can-drop {
```
Comment (marwa7med) [inline public/css/style.css]: please remove your comments

File: src/controllers/index.js [hunk 3]
```diff
@@ -3,7 +3,9 @@ const express = require('express');
 const router = express.Router();
 const home = require('./home');
 const levels = require('./levels');
+const game = require('./game');
 
-router.get('/', home.get);
-router.get('/levels', levels.get);
+router.get('/', home.get)
```
Comment (marwa7med) [inline src/controllers/index.js:7]: you can write
 module.exports=router
.get('/', home.get)
.get('/levels', levels.get)
.get('/game', game.get);

File: public/js/homeDom.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+const navButton = document.querySelector('.burger-btn'); // eslint-disable-line
```
Comment (marwa7med) [inline public/js/homeDom.js]: what is burger-btn 🍔

**Review by Alaa-Khattab — APPROVED** [review]

**Review by marwa7med — APPROVED** [review]

**Review by NadiaKhatib — APPROVED** [review]

File: public/js/gameDom.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+const panelButton = document.getElementById('levels'); // eslint-disable-line
+panelButton.addEventListener('click', () => {
+  const panel = document.querySelector('.panel'); //eslint-disable-line
+  if (panel.style.display === 'none') {
+    panel.style.display = 'block';
+  } else {
+    panel.style.display = 'none';
+  }
+});
```
Comment (Alaa-Khattab) [inline public/js/gameDom.js:9]: why you disabled all these lines ?

**Review by Alaa-Khattab — CHANGES_REQUESTED** [review]

**Review by marwa7med — COMMENTED** [review]
some of these will change

**Review by marwa7med — COMMENTED** [review]
i  will make some changing here

**Comment by MarwaBj** [comment]
i show all the changes i'm waiting @ismail2009 to make a change

File: src/controllers/index.js (hunk 3 above)
Comment (marwa7med) [reply src/controllers/index.js:7]: like @mhmdshorafa  teach us :)

**Review by marwa7med — CHANGES_REQUESTED** [review]

**Review by Alaa-Khattab — APPROVED** [review]

**Review by marwa7med — APPROVED** [review]

**Pull request merged**